From 43c7975cf3e501e062bb29ce0449f4b1db699e9b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Horia=20Geant=C4=83?= <horia.geanta@nxp.com>
Date: Fri, 25 Mar 2016 10:37:42 +0200
Subject: [PATCH 1252/1383] clocksource: arch_timer: export
 arm_arch_timer_reread symbol
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The symbol needs to be accesible by modules, for example tcrypt,
thus export it:

ERROR: "arm_arch_timer_reread" [crypto/tcrypt.ko] undefined!
make[1]: *** [__modpost] Error 1
make: *** [modules] Error 2
make: *** Waiting for unfinished jobs....

Fixes: 344ca1046f768 ("clocksource: arm_arch_timer: Timer reread erratum")
Signed-off-by: Horia GeantÄƒ <horia.geanta@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/clocksource/arm_arch_timer.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clocksource/arm_arch_timer.c b/drivers/clocksource/arm_arch_timer.c
index 6e61e2d..7ea49b0 100644
--- a/drivers/clocksource/arm_arch_timer.c
+++ b/drivers/clocksource/arm_arch_timer.c
@@ -75,6 +75,7 @@ static __always_inline u32 arch_timer_reg_read(int access,
 
 #ifndef arm_arch_timer_reread
 bool arm_arch_timer_reread;
+EXPORT_SYMBOL(arm_arch_timer_reread);
 #endif
 
 /*
-- 
2.8.1

